docs/source/02_get_started/04_new_project.md

```diff
@@ -35,13 +35,12 @@ The output lists the directory in which to find the project.
 
 You can create a new project from a configuration file if you prefer. The file must contain:
 
+-   `output_dir` The path in which to create the project directory. For example, the path can be `~` for your home directory, or `.` for the current working directory.
```

Comment:
I'm not too keen on this -- the intent is to keep the list looking like the alternative approach, of answering questions, then also specify the output directory. Putting this first doesn't particularly assist the reader (and adds verbosity to the list).

Reply:
I think it is more important for the list to match the example `config.yml` file provided, rather than matching the previous interactive example.

Separate to the question of whether `output_dir` should be in the list or not, what do you think about removing the duplicate (and hence redundant) sentence '`output_dir` can be set to `~` for home directory, or `.` for the current working directory.'

Reply:
There's a good point here that we don't explain what `output_dir` is for the `config.yml` and it does appear in the example. It makes the list look like it's missing something, and it's a bit confusing that you have to find this information below the list. 

I also thought it was weird that we never mention why you don't have to specify this `output_dir` when you don't have to answer questions.

Is there a happy medium to not have it so verbose but still mention it inline on the list?

Reply:
I've made an adjustment, see what you think. It reduces the verbosity in the list and explains the path further in the copy below.